CHANGELOG.adoc

```diff
@@ -6,17 +6,43 @@
 This document provides a high-level view of the changes introduced in {project-name} by release.
 For a detailed view of what has changed, refer to the {uri-repo}/commits/master[commit history] on GitHub.
 
-== _Unreleased_
+== 0.3.0 - <releaseDate>
+
+A special BlackHat USA Arsenal 2019 release!
+
+* https://docs.google.com/presentation/d/17P_l2n-hgCehQ5eTWilru4IXXHnGIRTj4ftoW4BiX5A/edit?usp=sharing[BlackHat USA Arsenal 2019 slides]
+
+
+=== Enhancements
+
+* Added Windows support ({uri-issue}129[#129])
+* Improved documentation for operation with Bettercap ({uri-issue}107[#107])
+* Added support for RDP v10.7 ({uri-issue}135[#135])
```

Comment:
This is misleading, since we dont support RDP 10.7 features, it just doesnt crash when the server announces that is wants to use RDP 10.7. I think it should go in bug fixes instead.